masquerade-maven-plugin/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.flipkart.rome</groupId>
+        <artifactId>masquerade</artifactId>
+        <version>0.1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>masquerade-maven-plugin</artifactId>
+    <packaging>maven-plugin</packaging>
```

Comment:
This is awesome to provide a mvn plugin (instead of just a java class and ask users to use generic exec-mvn-plugin). This being a library, it should work for all build tools like gradle too (assuming users of this library could be on any build tool). Hope for other build tool use cases, its trivial to integrate masquerade in their build cycle.